lib/video/postprocessing/visualmetrics/visualMetrics.js

```diff
@@ -16,9 +16,29 @@ const SCRIPT_PATH = path.join(
   'visualmetrics.py'
 );
 
+const PORTABLE_SCRIPT_PATH = path.join(
+  __dirname,
+  '..',
+  '..',
+  '..',
+  '..',
+  'browsertime',
+  'visualmetrics-portable.py'
+);
+
+async function getScript(options) {
```

Comment:
FYI I changed this, that do not need to be async since we are not returning a promise or doing anything async.